input/script_utils.go

```diff
@@ -1033,6 +1165,40 @@ func CommitScriptToRemoteConfirmed(key *btcec.PublicKey) ([]byte, error) {
 	return builder.Script()
 }
 
+// LeaseCommitScriptToRemoteConfirmed constructs the script for the output on
```

Comment:
We should extend the table driven tests in `input/script_utils.go` to exercise all the new witness generation functions added.

Reply:
Done!